doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -279,6 +279,54 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
+
+KRKNWK-17535: The application core can crash on nRF5340 after the OTA firmware update finishes if the factory data module is enabled.
+  In the initialization method of the factory data module, the factory data partition and a part of the application image is restricted by Fprotect.
+  That makes it impossible to confirm the new image in the Matter thread and the confirmation must be performed before the factory data module is initialized.
+
+  **Affected platforms:** nRF5340
+
+  **Workaround:** Manually cherry-pick and apply commit with fix to ``sdk-connectedhomeip`` (commit hash: ``TODO``).
```

Comment:
```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``TODO``).
```
I presume the DNM label is for getting the hash.

Does the user need to cherry-pick in addition to the below instructions? Or is it one or the other?

Reply:
Yes, indeed DNM label is for hash. The user must do all those steps to fix the issue, so cherry-pick the commit to sdk-connectedhomeip, then apply the fix to ota_util.c and ota_util.h, and the last thing, add the lines to the app_task.cpp file.

Reply:
I'd recommend adding explicit instructions here, then. 
```suggestion
  **Workaround:** Complete the following steps:
  
  1. Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``TODO``).
```
Then add #. to all the lines that start "Add the following", and remember to indent the code-blocks accordingly.